src/index.tsx

```diff
@@ -288,13 +294,13 @@ export class App extends React.Component<any, AppState> {
       elements = clearSelection(elements);
       this.forceUpdate();
       this.setState({ elementType: "selection" });
-      if (window.document.activeElement instanceof HTMLElement) {
-        window.document.activeElement.blur();
-      }
```

Comment:
This must stay here. It applies only to `ESC`. If you put in outside this branch, it 1) prevent typing into input elements, and 2) it doesn't blur an input element on esc.